app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -102,3 +101,95 @@ th .sort-arrow {
     vertical-align: middle;
     color: #fff;
 }
+
+.table-filters {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 100%;
+    padding: 10px 0;
+    margin-bottom: 10px;
+}
+
+.filter-controls {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+}
+
+.orcid-switch {
+    display: flex;
+    align-items: center;
+}
+
+.orcid-toggle-label {
+    margin-right: 10px;
+    font-size: 14px;
+    color: #333;
+    white-space: nowrap;
+    font-weight: lighter;
+}
+
+.switch {
+    position: relative;
+    display: inline-block;
+    width: 60px;
+    height: 30px;
+}
+
+.switch input {
+    opacity: 0;
+    width: 0;
+    height: 0;
+    display: none;
+}
+
+.slider {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    width: 100%;
+    height: 100%;
+    border-radius: 34px;
+    transition: background-color 0.4s;
+    background-color: #ccc;
+    cursor: pointer;
+    position: relative;
```

Comment:
If we remove `display: none` from the input as above, we'll have to likely change this to `position: absolute` and maybe remove `display: flex` so the slider goes over the input element